collectors/job.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collectors
+
+import (
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/net/context"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api"
+	v1batch "k8s.io/client-go/pkg/apis/batch/v1"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	descJobInfo = prometheus.NewDesc(
+		"kube_job_info",
+		"Information about job.",
+		[]string{"namespace", "job", "parallelism", "completions", "active_deadline_seconds"}, nil,
```

Comment:
parallelism, completions and active_deadline_seconds should be their own metrics